heartbeat/docs/heartbeat-observer-options.asciidoc

```diff
@@ -0,0 +1,7 @@
+[[configuration-observer-options]]
+== Specify Observer and Geo Options
+
+It is often useful to view and query various attributes of the instance Heartbeat is running on. This data is attached to events via the <<add-observer-metadata,`add_observer_metadata`>> processor. This processor populates the ECS `observer.*` fields. One field of particular utility is the `observer.geo.name` field, which you can configure via this processor. Use this field to create distinctive geographic regions for your uptime checks. You might label your regions by datacenter name or geographic region, e.g. `virginia-dc-1`, or `us-east-1a`, or `virginia-us`.
+
+Note that Heartbeat should not be used with the similarly named <<add-host-metadata,`add_host_metadata`>> processor. In ECS parlance the host is the thing that is monitored, not the thing doing the monitoring, which is the observer.
```

Comment:
> In ECS parlance the host is the thing that is monitored, not the thing doing the monitoring, which is the observer.

I find this sentence a little opaque, but I don't think it's worth holding up the change. Perhaps if we retool the second part as a separate thought it would be improved. This seems subjective to me, but perhaps a tech writer would be a better judge.

```
In ECS parlance the host is the thing that is monitored. The agent doing the monitoring is referred to as the observer.
```